FEM/rf_st_new.cpp

```diff
@@ -1858,26 +1830,20 @@ void CSourceTerm::FaceIntegration(CRFProcess* pcs, std::vector<long> const& node
 	int count;
 	double fac = 1.0;
 	CElem* face = new CElem(1);
-	// face->SetFace();
-	for (i = 0; i < (long)vec_possible_elements.size(); i++)
+	for (i = 0; i < (long)msh->ele_vector.size(); i++)
 	{
-		elem = msh->ele_vector[vec_possible_elements[i]];
-		if (!elem->GetMark())
```

Comment:
why is 'if (!elem->GetMark())' not necessary?

Reply:
As explained above, It is to  enable all elements to be searched.